server/app.ts

```diff
@@ -83,6 +84,9 @@ export default function createApp(
   app.get('/privacy-policy', (req, res) => res.render('pages/privacy-policy/privacy-policy'))
   app.use('/start', setupLegalSenderStartPage())
   app.use('/contact-helpdesk', setupContactHelpdesk(zendeskService))
+  app.use('/legal-sender/sign-out', (req, res) =>
```

Comment:
Create a new sign-out route that simply redirects to the correct route (with `&force=true` appended)